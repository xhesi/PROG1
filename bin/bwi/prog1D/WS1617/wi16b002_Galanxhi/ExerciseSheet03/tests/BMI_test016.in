40
1.60
