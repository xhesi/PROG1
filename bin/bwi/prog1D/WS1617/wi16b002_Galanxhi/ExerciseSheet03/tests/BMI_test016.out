weight [kg]: height [m]: m=40.00kg h=1.60m -> BMI=15.62 (Severely underweight)
